readme.md

```diff
@@ -0,0 +1,23 @@
+# Movie-Tracker/Library/Db
+
+## About the Project
+Our Movie/Film Management App is the perfect solution to organize your personal film library and keep track of your favorite movies. With this user-friendly application, you can effortlessly capture, manage, and discover all your movies. Never miss the title of a movie or the names of the lead actors or directors whose works you'd like to watch again. Our app makes it easy to enjoy your passion for movies and maintain a well-organized library. Discover and enjoy movies in a new way - with our Movie App!
+
+## Live Demo
+Visit our app in action on [Render](https://app-5yds.onrender.com)!
+
+## Technologies
+- **Frontend**: TypeScript
+- **Backend**: Java
+- **Infrastructure**: Docker, Deployed on [Render.com](https://render.com)
+
+## Team Members
+- [Alex](https://github.com/Hirnstromwelle)
+- [Dominic](https://github.com/makitango)
+- [Fabian](https://github.com/FabianMariusLink)
+
+## Getting Started
+
+Run `mvn dependency:resolve` to resolve dependencies with Maven.
+Install all npm dependencies with `npm install`.
+To run the app in development mode, use `npm run dev`, then open http://localhost:3000 to view it in the browser.
```

Comment:
When I run `mvn dependency:resolve` nothing happens on my machine! ;-) 

Recommend adding the `clone the repo` info first ;-)

Reply:
Do I not have to start the backend as well?

Reply:
How do I handle the mongoDB?

Reply:
Our suggestion: we delete the text, and you can use the deployment on render.